app/assets/javascripts/challenges.js

```diff
@@ -1,2 +1,12 @@
 // Place all the behaviors and hooks related to the matching controller here.
 // All this logic will automatically be available in application.js.
+
+// Purpose: Provides show and hide feature to a clickable element with "toggler" class
+// Date: 6/16/2020
+
+$(document).ready(function() {
+    $(".toggler").click(function(e) {
+        e.preventDefault();
+      $(".hide").toggle("hide");
```

Comment:
`.toggle("hide")` is really slow with a lot of teams in the DB, toggle without `"hide"` seems to be a lot faster.